src/sentry/models/scheduledjob.py

```diff
@@ -1,10 +1,29 @@
 from __future__ import absolute_import, print_function
 
-from django.db import models
+from django.db import models, transaction
 from django.utils import timezone
 from jsonfield import JSONField
 
-from sentry.db.models import Model, sane_repr
+from sentry.db.models import (BaseManager, Model, sane_repr)
+
+
+class ScheduledJobManager(BaseManager):
+    def schedule_job(self, payload, job):
```

Comment:
Yeah, I'd say the same thing. When we talked IRL, I mentioned just adding a top level function just to simplify this interface. Adding a manager doesn't really do that. You just simply trade off `ScheduledJobs.objects.bulk_create` for `ScheduledJobs.objects.schedule_job`. We also don't really need two functions here. We can just detect if `jobs` is a list or a string to see if we're scheduling more than 1 thing or not.